lib/init.coffee

```diff
@@ -8,22 +8,34 @@ module.exports =
       type: 'string'
       default: 'pydocstyle'
       description: "Path to executable pydocstyle cmd."
+      order: 1
+    addSelectCodes:
+      type: 'string'
```

Comment:
This might work better as an [`array`](https://atom.io/docs/api/v1.10.2/Config#array) since it will handle stripping any whitespace between the `,` for you.